services/src/main/java/org/keycloak/services/resources/admin/RealmAdminResource.java

```diff
@@ -359,6 +359,7 @@ public RealmRepresentation getRealm() {
 
             RealmRepresentation rep = new RealmRepresentation();
             rep.setRealm(realm.getName());
+            rep.setRegistrationEmailAsUsername(realm.isRegistrationEmailAsUsername());
```

Comment:
```suggestion
            if (auth.users().canView()) {
                rep.setRegistrationEmailAsUsername(realm.isRegistrationEmailAsUsername());
            }
```

Reply:
I am unresolving as it seems that in latest version there is this code:
```
            rep.setRegistrationEmailAsUsername(realm.isRegistrationEmailAsUsername());

            if (auth.users().canView()) {
                rep.setRegistrationEmailAsUsername(realm.isRegistrationEmailAsUsername());
            }
```
which clearly looks like a mistake (the very first line should be removed right?)